runtime/src/bank.rs

```diff
@@ -6300,10 +6300,21 @@ impl Bank {
     }
 
     pub(crate) fn shrink_ancient_slots(&self) {
-        self.rc
+        let can_skip_rewrites = self.bank_hash_skips_rent_rewrites();
+        let test_skip_rewrites_but_include_in_bank_hash = self
+            .rc
             .accounts
             .accounts_db
-            .shrink_ancient_slots(self.epoch_schedule())
+            .test_skip_rewrites_but_include_in_bank_hash;
```

Comment:
why are we using symbol names that contain the string "test" in prod?

Reply:
I didn’t introduce the field in this backport, only bind a local variable to the existing field preserving the name. As to why the original field with the name starting with the string test ended up in production, I don’t have an answer. What do you recommend should be changed in this backport?

Reply:
`test_skip_rewrites_but_include_in_bank_hash` is connected to a cli arg. It allows us to have a validator simulating skipping rewrites while still including them in the delta hash.